cfme/fixtures/service_fixtures.py

```diff
@@ -27,8 +27,18 @@ def catalog():
 
 @pytest.fixture(scope="function")
 def catalog_item(provider, provisioning, vm_name, dialog, catalog):
+    catalog_item = create_catalog_item(provider, provisioning, vm_name, dialog, catalog)
+    return catalog_item
+
+
+def create_catalog_item(provider, provisioning, vm_name, dialog, catalog, console_template,
+        use_console_template=None):
+
     template, host, datastore, iso_file, catalog_item_type, vlan = map(provisioning.get,
         ('template', 'host', 'datastore', 'iso_file', 'catalog_item_type', 'vlan'))
+    if use_console_template:
```

Comment:
You could get rid of the ``use_console_template`` parameter, define ``console_template=None`` and when the ``console_template`` is passed (is other than ``None``), the corresponding template is used.